version.sbt

```diff
@@ -0,0 +1,6 @@
+// NOTE: This is replaced with `sed` during release,
+// but it could also be removed, and be determined
+// based on `git` tags by https://github.com/dwijnand/sbt-dynver,
+// which is a dependency of `sbt-ci-release`.
+
+version in ThisBuild := "3.2.1-SNAPSHOT"
```

Comment:
won't this result in having the libraries locked with the version of mantis?

Reply:
It will, yeah. I think that at least is easy to understand in the context of a mono-repo, although it obscures semantic versioning of these libraries. If we wanted the version them separately then probably it would be better to move them out of Mantis into their own repository. Then we could strip them of their Akka dependency, enhance the API of the RLP type classes to not raise exceptions. Right now we just want to make a stop to copy pasting the RLP code between projects, and just ride the coattails of Mantis. Happy to hear your thoughts on versioning in the long term.

Reply:
I'll keep in mind that we should make this less dependent, thanks